src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts

```diff
@@ -64,17 +51,20 @@ export class CreateClusterReviewComponent implements OnInit {
       ]
     };
 
-    this.hostService.list().subscribe((resp: object[]) => {
+    this.hostService.list('true').subscribe((resp: object[]) => {
       this.hosts = resp;
       this.hostsCount = this.hosts.length;
 
       _.forEach(this.hosts, (hostKey) => {
         const labels = hostKey['labels'];
+        this.totalCPUs = this.totalCPUs + hostKey['cpu_count'];
+        this.totalMemory = this.totalMemory + hostKey['memory_total_kb'] * 125;
```

Comment:
Converting to bytes(B)

Reply:
will change name to `totalMemoryBytes` to make it more clear.

Reply:
Oh, you're assuming that this `_kb` there stands for kilobits? No, that's probably not formally correct (it should be uppercase `B` and `Ki` for the kibi prefix), but storage/memory is almost never measured in bits (only serial link speeds/bandwidths) and you can assume that `k` means 99% of the time `Kibi` (x1024) not strictly `kilo` (x1000).

In case of doubt you may check where this data was gathered from but I'm mostly sure that this is in 1024 x byte units.

Reply:
I assumed here tht from backend kb is kilobits so I'm converting it to KiB in UI. I can't verify if it's bits/bytes in backend.